modules/test/playwright/tests/frontend-data-set-views-web/pages/FDSFragmentPage.ts

```diff
@@ -29,6 +38,62 @@ export class FDSFragmentPage {
 		await this.page.goto('/');
 	}
 
+	async configureDataSetFragment({
```

Comment:
It is a wrapper of the steps needed to drag and drop the fragment.
My idea is to move them to private methods once the refactor of the `itemActions.spec` and `creationActions.spec` have been refactored to fix flakyness.